src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -114,24 +142,39 @@ export class ArtistInfo extends Component<ArtistInfoProps> {
                   />
                 </>
               )}
+              {showArtistInsightsButton && (
```

Comment:
Should the button disappear after having been clicked?

Reply:
The desire is to have it remain with an updated message (show vs. hide).